public/locales/en/translation.json

```diff
@@ -75,6 +75,10 @@
         "header": "INFORMATION COLLECTED FROM WEB TRAFFIC REPORTING TOOLS",
         "content": "When a GitHub Pages site is visited, the visitor's IP address is logged and stored for security purposes, regardless of whether the visitor has signed into GitHub or not. For more information about GitHub's security practices, see <a target=\"top\" href=\"https://docs.github.com/en/github/site-policy/github-privacy-statement\" class=\"dotcom-only\">GitHub Privacy Statement</a>."
       },
+      {
+        "header": "WEB TRACKING SERVICES",
```

Comment:
This should be included in the 'INFORMATION COLLECTED FROM WEB TRAFFIC REPORTING TOOLS' header, rather than a new header. Also, where did this text come from? Is there some boilerplate provided by Google analytics we can reuse?

Reply:
I couldn't find any boilerplate text to copy over. Google does have the following language in one of their [Help Center articles](https://support.google.com/analytics/answer/2700409?hl=en):

> If you've enabled any Google Analytics Advertising features, you are required to notify your visitors by disclosing the following information in your privacy policy:

> - The Google Analytics Advertising Features you've implemented.
> - How you and third-party vendors use first-party cookies (such as the Google Analytics cookie) or other first-party identifiers, and third-party cookies (such as Google advertising cookies) or other third-party identifiers together.
> - How visitors can opt-out of the Google Analytics Advertising Features you use, including through Ads Settings, Ad Settings for mobile apps, or any other available means (for example, the NAI's consumer opt-out). 

The text I wrote was primarily based on privacy policy statements I found from other sites that use Google Analytics,  mainly from that of Indeed.com which is described in this blog post:
https://www.termsfeed.com/blog/privacy-policy-google-analytics/#Disclose_The_Use_Of_Google_Analytics_In_Your_Privacy_Policy

I'm open to suggestions if you would like to modify the language.

Reply:
I moved the language to the 'INFORMATION COLLECTED FROM WEB TRAFFIC REPORTING TOOLS' header in the following commit: https://github.com/dvci/shc-web-verifier/pull/59/commits/fcd77df66dea759841af80e91956f53b16946984